cvat-ui/src/components/annotation-page/standard3D-workspace/styles.scss

```diff
@@ -171,3 +171,96 @@
 .cvat-control-side-bar-icon-size {
     font-size: $grid-unit-size * 5;
 }
+
+.cvat-canvas3d-perspective {
+    height: 50%;
+    width: 100%;
+    position: relative;
+    padding: $grid-unit-size/2;
+}
+
+.cvat-canvas3d-orthographic-views {
+    height: 50%;
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+}
+
+.cvat-canvas3d-perspective-arrow-directions {
+    position: absolute;
+    padding: $grid-unit-size;
+    bottom: 0;
+    right: 0;
+}
+
+.cvat-canvas3d-perspective-arrow-directions-icons-up {
+    margin-left: $grid-unit-size * 5.25;
+}
+
+.cvat-canvas3d-perspective-arrow-directions-icons-bottom {
+    margin: $grid-unit-size/2;
+}
+
+.cvat-canvas3d-perspective-arrow-directions-icons-color {
+    color: black;
+}
+
+.cvat-canvas3d-perspective-directions {
```

Comment:
Same as .cvat-canvas3d-perspective-arrow-directions. Use one common please

Reply:
There is one difference between positions left and right in both the css.